permissions/plugin-declarative-pipeline-migration-assistant.yml

```
@@ -6,6 +6,7 @@ issues:
 paths:
   - "org/jenkins-ci/plugins/to-declarative/declarative-pipeline-migration-assistant"
 developers:
+- "@cloudbees-developers"
```

this looks incorrect YAML
```suggestion
  - "@cloudbees-developers"
```

---

addressed in addd22728a8accbc679041d3116d0925cf72d248